src/main/java/Barry.java

```diff
@@ -0,0 +1,322 @@
+                    String deadlineDateTime = deadlineMatcher.group(2);
+                    taskList.add(new Deadline(deadlineName, deadlineDateTime));
+                    int listSize = taskList.size();
+                    String plural = listSize > 1 ? "s" : "";
+                    System.out.println("\t Gotcha, i've added the Deadline:");
+                    System.out.println("\t   " + taskList.get(listSize - 1).detailsAsString());
+                    System.out.println("\t You've now got " + listSize + " task" + plural + " in your list!");
+                } else {
+                    // Invalid format
+                    System.out.println("\t I see you're trying to add a Deadline, but it's in the wrong format...");
+                    System.out.println("\t [Type 'help' if you're unsure of the correct format]");
+                }
+                System.out.println("\t____________________________________________________________\n");
+            } else if (inputText.startsWith("event")) {
+                // Events
+                Matcher eventMatcher = eventPattern.matcher(inputText);
+                System.out.println("\t____________________________________________________________");
+                if (eventMatcher.matches()) {
+                    // Valid format
+                    String eventName = eventMatcher.group(1);
+                    String eventStartDateTime = eventMatcher.group(2);
+                    String eventEndDateTime = eventMatcher.group(3);
+                    taskList.add(new Event(eventName, eventStartDateTime, eventEndDateTime));
+                    int listSize = taskList.size();
+                    String plural = listSize > 1 ? "s" : "";
+                    System.out.println("\t Gotcha, i've added the Event:");
+                    System.out.println("\t   " + taskList.get(listSize - 1).detailsAsString());
+                    System.out.println("\t You've now got " + listSize + " task" + plural + " in your list!");
+                } else {
+                    // Invalid format
+                    System.out.println("\t I see you're trying to add an Event, but it's in the wrong format...");
+                    System.out.println("\t [Type 'help' if you're unsure of the correct format]");
+                }
+                System.out.println("\t____________________________________________________________\n");
+            } else {
+                // No such command
+                System.out.println("\t____________________________________________________________");
+                System.out.println("\t I'm sorry, I don't understand what that means... >_<");
+                System.out.println("\t [Type 'help' for a list of available commands]");
+                System.out.println("\t____________________________________________________________\n");
+            }
+        }
+    }
+
+    public static void main(String[] args) {
+        greet();
+        taskListMode();
```

Comment:
I like that you main method does not have much lines, but I think you could consider creating other class, using packages to structure your files better as well, especially in the taskListMode() method where it looks a bit clunky, I think you consider create perhaps a Command package and add the different command classes. Overall, I like the idea here and, keep up the good work!